<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightMode</class>
 <widget class="QWidget" name="FlightMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="paramsLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Fade In</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="fadeIn">
       <property name="decimals">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="fadeOut">
       <property name="decimals">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fade Out</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name">
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6" rowspan="2">
      <spacer name="spacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Switch</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="swtch"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="trimsLayout">
     <item row="0" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QSlider" name="trim2Slider">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="trim3Slider">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="leftTrims_3" columnstretch="0,1">
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_5c_3">
            <item>
             <spacer name="verticalSpacer_3c_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="trim2Use"/>
            </item>
            <item>
             <widget class="QSpinBox" name="trim2Value">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>-896</number>
              </property>
              <property name="maximum">
               <number>896</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4a_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_6b_3">
            <item>
             <spacer name="verticalSpacer_5b_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="trim1Use"/>
            </item>
            <item>
             <widget class="QSpinBox" name="trim1Value">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>-896</number>
              </property>
              <property name="maximum">
               <number>896</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_6b_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="trim2Label">
            <property name="text">
             <string notr="true">trim2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="trim1Label">
            <property name="text">
             <string notr="true">trim1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="trim4Slider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="trim1Slider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="rightTrims_2" columnstretch="0,1">
          <item row="1" column="0">
           <widget class="QLabel" name="trim4Label">
            <property name="text">
             <string notr="true">trim4</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_8d_2">
            <item>
             <spacer name="verticalSpacer_9d_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="trim4Use"/>
            </item>
            <item>
             <widget class="QSpinBox" name="trim4Value">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>-896</number>
              </property>
              <property name="maximum">
               <number>896</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_10c_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_7b_2">
            <item>
             <spacer name="verticalSpacer_7b_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="trim3Use"/>
            </item>
            <item>
             <widget class="QSpinBox" name="trim3Value">
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>-896</number>
              </property>
              <property name="maximum">
               <number>896</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_8d_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="trim3Label">
            <property name="text">
             <string notr="true">trim3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="2">
      <layout class="QGridLayout" name="t6Layout" columnstretch="0,1">
       <item row="2" column="0">
        <widget class="QLabel" name="trim6Label">
         <property name="text">
          <string>trim6</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="trim6Use"/>
         </item>
         <item>
          <widget class="QSpinBox" name="trim6Value">
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>-896</number>
           </property>
           <property name="maximum">
            <number>896</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="2">
      <widget class="QSlider" name="trim6Slider">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <layout class="QGridLayout" name="t8Layout" columnstretch="0,1">
       <item row="2" column="0">
        <widget class="QLabel" name="trim8Label">
         <property name="text">
          <string>trim8</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QComboBox" name="trim8Use"/>
         </item>
         <item>
          <widget class="QSpinBox" name="trim8Value">
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>-896</number>
           </property>
           <property name="maximum">
            <number>896</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="2">
      <widget class="QSlider" name="trim8Slider">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="trim5Slider">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QGridLayout" name="t5Layout" columnstretch="0,1">
       <item row="1" column="0">
        <widget class="QLabel" name="trim5Label">
         <property name="text">
          <string>trim5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QComboBox" name="trim5Use"/>
         </item>
         <item>
          <widget class="QSpinBox" name="trim5Value">
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>-896</number>
           </property>
           <property name="maximum">
            <number>896</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <widget class="QSlider" name="trim7Slider">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QGridLayout" name="t7Layout" columnstretch="0,1">
       <item row="1" column="0">
        <widget class="QLabel" name="trim7Label">
         <property name="text">
          <string>trim7</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QComboBox" name="trim7Use"/>
         </item>
         <item>
          <widget class="QSpinBox" name="trim7Value">
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>-896</number>
           </property>
           <property name="maximum">
            <number>896</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>fadeIn</tabstop>
  <tabstop>swtch</tabstop>
  <tabstop>fadeOut</tabstop>
  <tabstop>trim2Slider</tabstop>
  <tabstop>trim2Use</tabstop>
  <tabstop>trim2Value</tabstop>
  <tabstop>trim1Use</tabstop>
  <tabstop>trim1Value</tabstop>
  <tabstop>trim1Slider</tabstop>
  <tabstop>trim3Use</tabstop>
  <tabstop>trim3Value</tabstop>
  <tabstop>trim4Use</tabstop>
  <tabstop>trim4Value</tabstop>
  <tabstop>trim4Slider</tabstop>
  <tabstop>trim3Slider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
